src/Bicep.Core.Samples/Files/Metadata_CRLF/main.bicep

```diff
@@ -0,0 +1,45 @@
+/* 
+  This is a block comment.
+*/
+
+// metadata with value
+meta myString2 = 'string value'
+meta myInt2 = 42
+meta myTruth = true
+meta myFalsehood = false
+meta myEscapedString = 'First line\r\nSecond\ttabbed\tline'
+
+// object value
+meta foo = {

```

Comment:
I know there's been some back-and-forth on this, but seeing it written down like this, I feel like I'm personally in favor of `metadata` instead of `meta` - it's a little longer, but it communicates its purpose much more clearly to the user, and matches the `@metadata` decorator. @alex-frankel can we have another discussion?

Reply:
Agreed. We discussed today and the consensus is we should switch back to `metadata`.